build/scripts/build.ps1

```diff
@@ -446,53 +442,11 @@ function Test-Special() {
         $bootstrapDir = Make-BootstrapBuild
         Exec-Block { & ".\build\scripts\test-determinism.ps1" -bootstrapDir $bootstrapDir } | Out-Host
     } 
-    elseif ($testPerfCorrectness) {
-        Test-PerfCorrectness
-    }
-    elseif ($testPerfRun) {
-        Test-PerfRun
-    }
     else {
         throw "Not a special test"
     }
 }
 
-function Test-PerfCorrectness() {
-    Run-MSBuild "Roslyn.sln" "/p:DeployExtension=$deployExtensions" -logFileName "RoslynPerfCorrectness"
-    Exec-Block { & ".\Binaries\$buildConfiguration\Exes\Perf.Runner\Roslyn.Test.Performance.Runner.exe" --ci-test } | Out-Host
```

Comment:
Are we deleting this too, or is it already deleted?

Reply:
Updated to delete